src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -42,32 +42,66 @@ public function getConfigTreeBuilder(): TreeBuilder
+                    ->useAttributeAsKey('prefix')
+                    ->arrayPrototype()
+                        ->info('the icon set prefix (e.g. "acme")')
+                        ->children()
+                            ->scalarNode('path')
+                                ->info("The local icon set directory path.\n(cannot be used with 'alias')")
+                                ->example('%kernel.project_dir%/assets/svg/acme')
+                            ->end()
+                            ->scalarNode('alias')
+                                ->info("The remote icon set identifier.\n(cannot be used with 'path')")
+                                ->example('simple-icons')
+                            ->end()
+                            ->arrayNode('icon_attributes')
+                                ->info('Override default icon attributes for icons in this set.')
+                                ->example(['class' => 'icon icon-acme', 'fill' => 'none'])
+                                ->normalizeKeys(false)
+                                ->variablePrototype()
+                                ->end()
+                            ->end()
+                        ->end()
+                    ->end()
+                    ->validate()
+                        ->ifTrue(fn (array $v) => isset($v['path']) && isset($v['alias']))
+                        ->thenInvalid('You cannot define both "path" and "alias" for an icon set.')
+                    ->end()
                 ->end()
                 ->arrayNode('aliases')
-                    ->info('Icon aliases (alias => icon name).')
-                    ->example(['dots' => 'clarity:ellipsis-horizontal-line'])
+                    ->info('Icon aliases (map of alias => full name).')
+                    ->example([
+                        'dots' => 'clarity:ellipsis-horizontal-line',
+                        'privacy' => 'bi:cookie',
+                    ])
                     ->normalizeKeys(false)
                     ->scalarPrototype()
                         ->cannotBeEmpty()
                     ->end()
                 ->end()
                 ->arrayNode('iconify')
-                    ->info('Configuration for the "on demand" icons powered by Iconify.design.')
+                    ->info('Configuration for the remote icon service.')
                     ->{interface_exists(HttpClientInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                     ->children()
                         ->booleanNode('on_demand')
-                            ->info('Whether to use the "on demand" icons powered by Iconify.design.')
+                            ->info('Whether to download icons "on demand".')
                             ->defaultTrue()
                         ->end()
                         ->scalarNode('endpoint')
-                            ->info('The endpoint for the Iconify API.')
+                            ->info('The endpoint for the iconify icons API.')
```

Comment:
```suggestion
                            ->info('The endpoint for the Iconify icons API.')
```

Reply:
Thanks i made changes, reverted, made other ... and in the end lost my way 😅